### PR TITLE
Fixes #25445: Document the 8.2 windows variable changes

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/25445

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

File: src/reference/modules/installation/pages/upgrade/notes.adoc [hunk 1]
```diff
@@ -26,92 +26,63 @@ rudder package list --all
...
+The generation safety is done by a very simple script located in a new `policy-generation-pre-start` webapp hook
+located in `/opt/rudder/etc/hooks.d/policy-generation-pre-start` and distributed with the Windows plugin.
 
-The support of user authentication through Radius, which was deprecated since Rudder 7, has been removed.
+Note that in some cases the hook may block the generation if you are using inputs similar to the now deprecated
+syntax. If it is the case, please try to use `${const.dollar}(...)` instead of `$(...)` in the directives/methods
+fields.
 
-=== API tokens
+The hook can still be deactivated by commenting its content if needed, with the risk of deploying potentially
```
Comment (fanf) [inline src/reference/modules/installation/pages/upgrade/notes.adoc:74]: you can just `set -x` it, too

Comment (Fdall) [reply src/reference/modules/installation/pages/upgrade/notes.adoc:74]: How is that gonna disable the hook ?

Comment (fanf) [reply src/reference/modules/installation/pages/upgrade/notes.adoc:74]: only executable hooks are executed by rudder

Comment (fanf) [reply src/reference/modules/installation/pages/upgrade/notes.adoc:74]: sorry, I wasn't not clear: I meant to say "you can just remove the executable flag on the hook with chmod -x", and mixed things up

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**